app/models/concerns/audit_search.rb

```diff
@@ -51,14 +51,15 @@ def auditable_type_complete_values
         :host => 'Host::Base',
         :interface => 'Nic::Base',
         :location => 'Location',
-        :os => 'Operatingsystem',
         :organization => 'Organization',
+        :os => 'Operatingsystem',
         :override_value => 'LookupValue',
+        :parameter => 'Parameter',
         :partition_table => 'Ptable',
+        :setting => 'Setting',
         :smart_class_parameter => 'PuppetclassLookupKey',
         :smart_variable => 'VariableLookupKey',
-        :parameter => 'Parameter',
-        :setting => 'Setting'
+        :subnet => 'Subnet'
```

Comment:
This is the only line added, others are just reordering the list alphabetically.